servo

```diff
@@ -97,8 +97,8 @@ def measure(param):
     rsp = run_driver(DFLT_MEASURE_DRIVER, args.app_id, req=param, progress_cb=partial(report_progress, 'MEASUREMENT', time.time()))
     status = rsp.get('status', 'undefined')
     if status != 'ok':
-        raise Exception('Measurement driver failed with status "{}" and message "{}"'.format(
-            status, rsp.get('message', 'undefined')))
+        raise DriverError('Measurement driver failed with status "{}" and message:\n{}'.format(
+            status, str(rsp.get('message', 'undefined'))), status=status, reason=rsp.get('reason', 'undefined'))
```

Comment:
What is the reason for using str() here? The value of rsp['message'] is expected to be a string, anyway. NOTE that if an adjust/measure driver sends something that is not a string, the run_driver() function will fail (that's a separate bug, though, no action required for this commit)

Reply:
According to the relevant PR (https://github.com/opsani/servo/pull/10), response message was a type whose __repr__ method produced undesirable results and needed to have the __str__ method used instead. 

Specifically it calls out "python binary repr" but I was unable to contextualize this because it shouldn't be possible for message to be a byte str and binary numbers are stored in the standard int type whose __repr__ returns base 10 even if instantiated with a binary literal.

@Johnius can you recall the adjust driver you were getting this behavior with? I just want to be sure removing the str invocation won't cause a regression

Reply:
@linkous8 I can't say I ever had this issue at all. Also, aren't we moving to ServoX altogether?

Reply:
I can't speak to the timeline of moving to servox but I've been asked to make some changes to this model for fast fail. Thanks getting back on this, I prefer to understand the reason code was added before I go deleting it and this puts my mind at ease :+1: